docs/flink-getting-started.md

```diff
@@ -553,14 +553,18 @@ env.execute("Test Iceberg DataStream");
 
 ## Write options
 
-Flink write options are passed when configuring the FlinkSink, like this:
+Flink write options are passed when configuring the FlinkSink or Flink SQL hints, like this:
 
 ```
 FlinkSink.Builder builder = FlinkSink.forRow(dataStream, SimpleDataUtil.FLINK_SCHEMA)
     .table(table)
     .tableLoader(tableLoader)
     .set("write-format", "orc")
     .set(FlinkWriteOptions.OVERWRITE_MODE, "true");
+    
+INSERT INTO talbeName  /*+ OPTIONS('upsert-enabled'='true')*/ 
+SELECT 
```

Comment:
nit: either complete `select` part or just leave it as `...`

Reply:
also it is probably more clear to start the SQL hints as a separate paragraph after the Java API code snippet. e.g.

For Flink SQL, write options can be passed as hints like this
```
INSERT INTO tableName  /*+ OPTIONS('upsert-enabled'='true')*/  
...
```

Reply:
Nit: Is it possible to have a space before the terminal `*/`? If not, no big deal but it would be nice if we could.